addons/survey/controllers/survey_session_manage.py

```diff
@@ -107,13 +111,20 @@ def survey_session_next_question(self, survey_token, go_back=False, **kwargs):
                 'session_question_id': next_question.id,
                 'session_question_start_time': fields.Datetime.now() + relativedelta(seconds=1)
             })
-            request.env['bus.bus']._sendone(survey.access_token, 'next_question', {'question_start': now.timestamp()})
+            request.env['bus.bus']._sendone(survey.access_token, 'next_question', {
+                'question_start': now.timestamp(),
+                'go_back': go_back,
```

Comment:
IIRC this was necessary for the previous "light" implementation that always faded when going back, right?

Is it needed anymore in this "very super light" version?

If not, there are a few lines to clean.